radix-engine-interface/src/blueprints/resource/bucket.rs

```diff
@@ -77,9 +77,31 @@ pub type BucketCreateProofOfAllOutput = Proof;
 // Stub
 //========
 
+// TODO: update schema type
```

Comment:
We could/should probably make this transparent too?

And we can derive `ScryptoCategorize`, `ScryptoEncode` and `ScryptoDecode` automatically? :) - (with Describe manual for now)